packages/ui-components/CHANGELOG.md

```diff
@@ -13,6 +13,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Renamed packages and scope to `@mechanic-design/ui-components`.
 
+### Fixed
+
+- Fixed color input bug that prevented it from closing when clicking another instance of the input, and a related z-index issue
+
+- Fixed inconsistencies on hover and active states across all inputs and buttons
+
```

Comment:
Move this under Unreleased since it's not jet under any release. Also add a similar log in root folder CHANGELOG.